packages/bpk-component-price/src/BpkPrice.js

```diff
@@ -40,15 +37,17 @@ type Props = {
 const getClassName = cssModules(STYLES);
 
 const BpkPrice = (props: Props) => {
-  const { className, description, size, subtitle, title, ...rest } = props;
+  const { align, className, description, size, subtitle, title, ...rest } =
+    props;
 
   const isSmall = size === SIZES.small;
+  const isAlighRight = align === ALIGNS.right;
```

Comment:
Minor spelling mistake
```suggestion
  const isAlignRight = align === ALIGNS.right;
```